foundry/entities/combat.d.ts

```diff
@@ -61,12 +61,12 @@ declare class Combat extends Entity<Combat.Data> {
   /**
    * Get the data object for the Combatant who has the current turn
    */
-  get combatant(): Combat.Combatant;
+  get combatant(): this['turns'][number];
 
   /**
    * A convenience reference to the Array of combatant data within the Combat entity
    */
-  get combatants(): Combat.Combatant[];
+  get combatants(): this['data']['combatants'];
```

Comment:
This directly returns `this.data.combatants` and that is an array of combatants before any processing is done. This change was done to specifically adress a bug that FloRad brought up in the discord. Although I think I need to go over `Expanded` again, because it should have worked.